deps/v8/gypfiles/toolchain.gypi

```diff
@@ -1393,7 +1393,7 @@
         ],  # conditions
       },  # Release
     },  # configurations
-    'cflags': [ '-Wno-type-limits', ],
+    'cflags': [ '-w', ],
```

Comment:
I've been considering this as well. Locally I played with
```python
    'cflags': [ '-Wno-type-limits', '-Wno-class-memaccess', ],
```
Maybe we should try:
```python
    'cflags!': [ '-Wall', '-Wextra' ],
```

Reply:
done